File: app/src/main/java/com/unicorn/sxmobileoa/app/network/model/FydmRequest.java
package com.unicorn.sxmobileoa.app.network.model;

import com.unicorn.sxmobileoa.app.Key;

public class FydmRequest extends request {

    public FydmRequest(String busiCode, String fydm) {
        super(busiCode);
        addParameter(Key.fydm, fydm);
    }

}

File: app/src/main/java/com/unicorn/sxmobileoa/app/network/model/FydmRequest.kt
package com.unicorn.sxmobileoa.app.network.model

import com.unicorn.sxmobileoa.app.Key

open class FydmRequest(busiCode: String, fydm: String) : request(busiCode) {

    init {
        addParameter(Key.fydm, fydm)
    }

}
